src/coreclr/jit/vartype.h

```diff
@@ -70,7 +70,17 @@ inline bool varTypeIsSIMD(T vt)
 #else
     // Always return false if FEATURE_SIMD is not enabled
     return false;
-#endif // !FEATURE_SIMD
+#endif
+}
+
+template <class T>
+inline bool varTypeIsMask(T vt)
```

Comment:
This should be under `TARGET_AMD64`?

Reply:
Here and at other places around `isMaskReg()`, etc.

Reply:
`TARGET_XARCH`, but yes we should try to keep the logic as non-impacting to Arm/Arm64.

Reply:
That's how we handle many other similar APIs. Looks like it was adjusted to do just that so I'll mark this thread as "resolved".